docs/setup-guide.mdx

```diff
@@ -2,53 +2,63 @@
 title: Setup Guide
 ---
 
-:::note
-This guide assumes you know how to use the command line. You can find tutorials on YouTube for [Windows](https://www.youtube.com/watch?v=FiTZgpRpWv0) or [MacOS](https://www.youtube.com/watch?v=aKRYQsKR46I).
+:::info[Prerequisites]
+- Familiarity with the command line. You can find tutorials on YouTube for [Windows](https://www.youtube.com/watch?v=FiTZgpRpWv0) or [MacOS](https://www.youtube.com/watch?v=aKRYQsKR46I)
+- [NodeJS 18+](https://nodejs.org/)
+- A code editor such as [VSCode](https://code.visualstudio.com/)
+- [Rojo 7+](https://rojo.space/) which can be installed through the [Rojo VSCode plugin](https://marketplace.visualstudio.com/items?itemName=evaera.vscode-rojo).
 :::
 
-## Before You Begin
-
-In order to start using roblox-ts, you'll need to have the following software already installed:
-- [NodeJS 14+](https://nodejs.org/)
-- A code editor. We recommend [VSCode](https://code.visualstudio.com/).
-- [Rojo 6+](https://rojo.space/). The easiest way to install it is through the [Rojo VSCode plugin](https://marketplace.visualstudio.com/items?itemName=evaera.vscode-rojo).
-
-## Project Setup
-
-To start using roblox-ts to create a game project, follow these steps:
-
-<!-- Unfortunately, we need to explicitly number these steps because there are images between them :( -->
+:::caution PowerShell
+If you're using PowerShell for the first time, you may get an error about "running scripts is disabled on this system". You can fix this by running the following in PowerShell as an administrator:
 
-1. Create a new folder and give it a name. The folder name cannot contain spaces. We are going to name ours `my-project`.
+`Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`
 
-<center><img src={require("../static/img/setup-guide/new-folder.png").default} /></center>
+[More info here](./articles/powershell.mdx)
+:::
 
-2. Open the folder using your code editor (or your command line if your code editor does not have one integrated).
+## Project Setup
 
-<center><img src={require("../static/img/setup-guide/open-with-vscode.png").default} /></center>
+In this guide, we will introduce how to create a roblox-ts game.
 
-3. Run `npm init roblox-ts` in your command line to start the interactive project setup.
+Make sure you have an up-to-date version of Node.js installed and your current working directory is the one where you intend to create a project. Run the following command in your command line:
```

Comment:
This sentence says the cwd should be the folder you intend to create a project in. It's ambiguous to me whether that means the project folder itself, or an outer one. It's also a bit double considering that step 1 also asks for a project directory. I would suggest clarifying here that it should be a fresh empty folder for the project, and advising to put `.` as the directory in step 1.